docs/UserGuide.md

```diff
@@ -64,6 +64,14 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 </div>
 
+### Instantaneous launching
+Users that have Java 11 or above installed in their computers can launch the Duke The Market program 
```

Comment:
Are we calling it Duke-The-Market?

Reply:
Feel free to refer to the conversation on #16.

_On a side note, is there a way to make this link to the direct conversation instead of the issue itself?_